Schema/PublicSchema/ContextTagKeys.bond

```diff
@@ -0,0 +1,105 @@
+    [MaxStringLength("256")]
+    140: string 	 DeviceOSVersion = "ai.device.osVersion";
+    
+    [Description("The type of the device the end user of the application is using. Used primarily to distinguish JavaScript telemetry from server side telemetry. Examples: 'PC', 'Phone', 'Browser'. 'PC' is the default value.")]
+    [MaxStringLength("64")]
+    160: string 	 DeviceType = "ai.device.type";
+    
+    [Description("The IP address of the client device. IPv4 and IPv6 is supported. Information in the location context fields is always about the end user. When telemetry is sent from a service, the location context is about the user that initiated the operation in the service.")]
+    [MaxStringLength("45")]
+    200: string 	 LocationIp = "ai.location.ip";
+    
+    [Description("A unique identifier for the operation instance. The operation.id is created by either a request or a page view. All other telemetry sets this to the value for the containing request or page view. Operation.id is used for finding all the telemetry items for a specific operation instance.")]
+    [MaxStringLength("128")]
+    300: string 	 OperationId = "ai.operation.id";
+    
+    [Description("The name (group) of the operation. The operation.name is created by either a request or a page view. All other telemetry items set this to the value for the containing request or page view. Operation.name is used for finding all the telemetry items for a group of operations (i.e. 'GET Home/Index').")]
+    [MaxStringLength("1024")]
+    305: string 	 OperationName = "ai.operation.name";
+    
+    [Description("The unique identifier of the telemetry item's immediate parent.")]
+    [MaxStringLength("128")]
+    310: string 	 OperationParentId = "ai.operation.parentId";
+    
+    [Description("Name of synthetic source. Some telemetry from the application may represent a synthetic traffic. It may be web crawler indexing the web site, site availability tests or traces from diagnostic libraries like Application Insights SDK itself.")]
+    [MaxStringLength("1024")]
+    320: string 	 OperationSyntheticSource = "ai.operation.syntheticSource";
+    
+    [Description("The correlation vector is a light weight vector clock which can be used to identify and order related events across clients and services.")]
+    [MaxStringLength("64")]
+    330: string 	 OperationCorrelationVector = "ai.operation.correlationVector";
+    
+    [Description("Session ID - the instance of the user's interaction with the app. Information in the session context fields is always about the end user. When telemetry is sent from a service, the session context is about the user that initiated the operation in the service.")]
+    [MaxStringLength("64")]
+    400: string 	 SessionId = "ai.session.id";
```

Comment:
Just curious, if this is about the user then why isn't it "ai.user.session.id"?
And how is it different than "ai.user.id"?